packages/@react-native-windows/telemetry/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@react-native-windows/telemetry",
+  "version": "0.63.1",
+  "license": "MIT",
+  "main": "lib-commonjs/index.js",
+  "typings": "lib-commonjs/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:microsoft/react-native-windows.git",
+    "directory": "packages/@react-native-windows/telemetry"
+  },
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "lint": "just-scripts lint",
+    "lint:fix": "just-scripts lint:fix",
+    "test": "set RNW_CLI_TEST=true&& just-scripts test",
+    "watch": "just-scripts watch"
+  },
+  "dependencies": {
+    "applicationinsights": "^1.8.8"
+  },
+  "devDependencies": {
+    "@rnw-scripts/eslint-config": "0.0.2",
+    "@rnw-scripts/jest-unittest-config": "0.1.1",
+    "@rnw-scripts/just-task": "0.0.2",
+    "@rnw-scripts/ts-config": "0.1.0",
+    "@types/jest": "^24.9.1",
+    "@types/semver": "^7.3.1",
+    "babel-jest": "^24.9.0",
+    "eslint": "6.7.0",
+    "jest": "^24.9.0",
+    "just-scripts": "^0.36.1",
+    "prettier": "1.18.2",
+    "typescript": "^3.8.3"
+  },
+  "files": [
+    "lib-commonjs"
+  ],
+  "beachball": {
+    "defaultNpmTag": "canary",
```

Comment:
Need to change `defaultNpmTag` to `latest` and switch `patch` for `prerelease` in `disallowedChangeTypes`. This is usually done by tooling when we cut a branch.

Reply:
We really really need to push node to allow defaults for package.json as well as support for config files so we don't have to have so many default config files forwardning in the root.....

We are forced into really poor engineering patterns where one needs "templates" rather than "defaults" to do ones job.

Reply:
For this one specifically we could also have a separate JS config file if we wanted to, but so far we've been just keeping Beachball config in each package.json.